spec/request/user/create_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe 'POST #create sign_up', type: :request do  
+  let!(:user_params) { { user: FactoryBot.attributes_for(:user) } }
+  subject do
```

Comment:
I think it would be better to leave an empty line after the `let`. Also, here in the specs you can use `attributes_for(:user)` without the `FactoryBot`.